internal/controlplane/handlers_githubwebhooks.go

```diff
@@ -326,7 +326,7 @@ func (s *Server) registerWebhookForRepository(
 	regResult.Repository = &pb.Repository{
 		Name:          repoGet.GetName(),
 		Owner:         repoGet.GetOwner().GetLogin(),
-		RepoId:        int32(repoGet.GetID()),
+		RepoId:        repoGet.GetID(),
```

Comment:
oops!!

Reply:
I created the private branch before #2415 merged, so this passed tests, and that passed tests, but the merge did not.  I honestly expected there would be a merge conflict that would protect, but clearly not.